notebooks/13_Theta_phase_and_power.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    lfp_key["target_interval_list_name"] = "test interval"
</pre><p>typo fix. Does "test interval" exist for this nwb file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
yep! And one always needs to make sure it exists for a given nwb file in order to do the analytic signal analyses.